src/test/regress/expected/gist.out

```diff
@@ -371,8 +371,7 @@ where p <@ box(point(5, 5), point(5.3, 5.3));
    ->  Index Only Scan using gist_tbl_multi_index on public.gist_tbl
          Output: circle(p, '1'::double precision)
          Index Cond: (gist_tbl.p <@ '(5.3,5.3),(5,5)'::box)
- Settings: enable_bitmapscan = 'off', enable_indexonlyscan = 'on', enable_seqscan = 'off'
```

Comment:
i saw CBDB have the `gist_optimizer.out`, the `gist.out` is the result of the `-c optimizer=off`, why ORCA is enabled in this case ?

Reply:
Thanks, most probably error, does not actual after rebase